Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -276,6 +278,7 @@ bool CruRawReader::parseDigitHCHeaders(int hcid)
   std::array<uint32_t, 3> headers{0};
 
   for (int headerwordcount = 0; headerwordcount < additionalHeaderWords; ++headerwordcount) {
+    LOGP(info, " HEADER : {} headers[] {}", headerwordcount, headers[headerwordcount]);
```

Comment:
remove debug output

Reply:
ja this push was mid changes, and then got distracted.